src/rgw/rgw_rest_s3.h

```diff
@@ -108,6 +108,8 @@ class RGWGetUsage_ObjStore_S3 : public RGWGetUsage_ObjStore {
 
 class RGWListBucket_ObjStore_S3 : public RGWListBucket_ObjStore {
 protected:  bool objs_container;
+virtual int get_common_params();
```

Comment:
virtual keyword is not required here, you can remove this.

Reply:
Removed.